main.tf

```diff
@@ -89,7 +89,7 @@ resource "azurerm_kubernetes_cluster" "aks" {
   default_node_pool {
     enable_node_public_ip = var.node_pools.platform.enable_node_public_ip
     name                  = "platform"
-    node_count            = var.node_pools.platform.max_count
+    node_count            = var.node_pools.platform.min_count
     node_labels           = merge({ "dominodatalab.com/node-pool" : "platform" }, var.node_pools.platform.node_labels)
     vm_size               = var.node_pools.platform.vm_size
     availability_zones    = var.node_pools.platform.zones
```

Comment:
It's off the diff by a line, but @Secretions do you recall offhand why the `max_pods` are overridden?

Reply:
No, I have to assume it was unintentional.